apps/prairielearn/src/lib/db-types.ts

```diff
@@ -924,3 +924,20 @@ export const GradingJobStatusSchema = z.enum([
   'requested',
 ]);
 export type GradingJobStatus = z.infer<typeof GradingJobStatusSchema>;
+
+export const JobSequenceSchema = z.object({
+  assessment_id: IdSchema.nullable(),
+  authn_user_id: IdSchema.nullable(),
+  course_id: IdSchema.nullable(),
+  course_instance_id: IdSchema.nullable(),
+  course_request_id: IdSchema.nullable(),
+  description: z.string().nullable(),
+  finish_date: DateFromISOString.nullable(),
+  id: IdSchema,
+  legacy: z.boolean(),
+  number: z.number().nullable(),
+  start_date: DateFromISOString.nullable(),
+  status: z.string().nullable(),
```

Comment:
This is actually an enum column, and in this case, we prefer to use an enum type, mostly for IDE completion.

For an example, see `EnumPlanGrantTypeSchema` here:

https://github.com/PrairieLearn/PrairieLearn/blob/ccf6649762c1c73ceeb2aa17efd686eca0d0df41/apps/prairielearn/src/lib/db-types.ts#L325

This is then used in `PlanGrantSchema` for the `type` column:

https://github.com/PrairieLearn/PrairieLearn/blob/ccf6649762c1c73ceeb2aa17efd686eca0d0df41/apps/prairielearn/src/lib/db-types.ts#L334

Reply:
Added `EnumJobStatusSchema` in [eaded80](https://github.com/PrairieLearn/PrairieLearn/pull/9734/commits/eaded80aaf1048e91535830298092335273e5901). Now using it here and in `JobSchema.status`.